project/Dependencies.scala

```diff
@@ -10,7 +10,7 @@ object Dependencies {
   val akkaVersion: String     = sys.props.getOrElse("akka.version", "2.5.26")
   val akkaHttpVersion: String = sys.props.getOrElse("akka.http.version", "10.1.11")
 
-  val sslConfig = "com.typesafe" %% "ssl-config-core" % "0.3.8"
+  val sslConfig = "com.typesafe" %% "ssl-config-core" % "0.4.0"
```

Comment:
```suggestion
  val sslConfig = "com.typesafe" %% "ssl-config-core" % "0.4.1"
```
Just realised `2.8.x` already uses `0.4.1`.